scripts/zones/Aht_Urhgan_Whitegate/Zone.lua

```diff
@@ -3,225 +3,225 @@
 -- Zone: Aht_Urhgan_Whitegate (50)
 --
 -----------------------------------
-local ID = require("scripts/zones/Aht_Urhgan_Whitegate/IDs");
-require("scripts/globals/settings");
-require("scripts/globals/keyitems");
-require("scripts/globals/missions");
-require("scripts/globals/quests");
-require("scripts/globals/status");
-require("scripts/globals/titles");
+local ID = require("scripts/zones/Aht_Urhgan_Whitegate/IDs")
+require("scripts/globals/settings")
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+require("scripts/globals/quests")
+require("scripts/globals/status")
+require("scripts/globals/titles")
 require("scripts/globals/zone")
 -----------------------------------
 
 function onInitialize(zone)
     zone:registerRegion(1,57,-1,-70,62,1,-65); -- Sets Mark for "Got It All" Quest cutscene.
```

Comment:
maybe a follow up regedit to catch ones with comments?

Reply:
Nice eye, pushed~